cmd/argo/commands/uninstall.go

```diff
@@ -76,4 +76,30 @@ func uninstall(cmd *cobra.Command, args []string) {
 	} else {
 		fmt.Printf("Workflow CRD '%s' deleted\n", wfv1.CRDFullName)
 	}
+
+	// Delete role binding
+	_, err = clientset.RbacV1beta1().ClusterRoleBindings().Get(ArgoClusterRole, metav1.GetOptions{})
```

Comment:
Can we switch from GET + DELETE into just a blind DELETE and treat IsNotFound as an acceptable error? The same goes for deleting the service account.